roles/horizon/templates/etc/openstack-dashboard/local_settings.py

```diff
@@ -53,6 +53,7 @@
 OPENSTACK_API_VERSIONS = {
 #    "data-processing": 1.1,
     "identity": 3,
+    "image": 1,
```

Comment:
Parameterize - IMAGES_ALLOW_LOCATION with default value of False    (this will hide url method in horizon).
https://github.com/blueboxgroup/ursula/blob/master/roles/horizon/templates/etc/openstack-dashboard/local_settings.py#L366

roles/horizon/defaults/main.yml
```
horizon:
  glance:
     allow_location: false
```
and use this in local_settings.py to replace current hardcoded value of True.